Makefile.work

```diff
@@ -103,6 +103,7 @@ DOCKER_BUILD = docker build --no-cache \
 
 SONIC_BUILD_INSTRUCTION :=  make \
                            -f slave.mk \
+                           BLDENV=$(BLDENV) \
```

Comment:
>BLDENV [](start = 27, length = 6)

slave.mk could detect the BLDENV from /etc/os-release. We should not treat it as a parameter.

Reply:
good suggestion, I'll try later.